new/detector/implementation/javascript/string/string.go

```diff
@@ -54,11 +53,15 @@ func concatenateChildren(node *tree.Node, evaluator types.Evaluator) ([]interfac
 			continue
 		}
 
-		childValue, childIsLiteral, err := getStringValue(child, evaluator)
+		childValue, childIsLiteral, err := generic.GetStringValue(child, evaluator)
 		if err != nil {
 			return nil, err
 		}
 
+		if childValue == "" {
```

Comment:
This should prevent empty string literals `""` being considered a variable:
```suggestion
		if childValue == "" && !childIsLiteral {
```